homeassistant/components/freebox/sensor.py

```diff
@@ -213,3 +241,52 @@ def async_update_state(self) -> None:
                 self._partition["free_bytes"] * 100 / self._partition["total_bytes"], 2
             )
         self._attr_native_value = value
+
+
+class FreeboxRaidDegradedSensor(BinarySensorEntity):
```

Comment:
We should not add a `BinarySensorEntity` to the `sensor` platform. Add this to `binary_sensor.py`.